src/main/java/net/starschema/clouddb/jdbc/BQStatement.java

```diff
@@ -117,6 +117,14 @@ public void setLabels(Map<String, String> statementLabels) {
         this.statementLabels = ImmutableMap.copyOf(statementLabels);
     }
 
+    @Override
+    public int getQueryTimeout() {
+        if (this.connection.getTimeoutMs() != null) {
+            return this.connection.getTimeoutMs().intValue();
+        }
+        return this.querytimeout * 1000;
```

Comment:
is `this.querytimeout` always set ?

Reply:
Yes, it's set as a constant in the parent class https://github.com/jonathanswenson/bqjdbc/blob/5ccf45b3f3cbf766a444bd25d83456cf88c047ee/src/main/java/net/starschema/clouddb/jdbc/BQStatementRoot.java#L71